discopop_explorer/pattern_detectors/task_parallelism/alias_detection.py

```diff
@@ -285,7 +285,8 @@ def get_alias_information(file_mapping: str, cu_xml: str, temp_file: str, build_
     if os.path.exists(temp_file):
         os.remove(temp_file)
     # create statements file
-    __create_statements_file(file_mapping, temp_file + "_statements", build_path + "/rtlib/simple-alias-detection/getStatements")
+    __create_statements_file(file_mapping, temp_file + "_statements",
+                             build_path + "/rtlib/simple-alias-detection/getStatements")
```

Comment:
plz use pathlib for joining directory paths, instead.

Reply:
should be fixed now